### PR TITLE
Adds `huey` for asynchronous data ingestion

### LINKED ISSUE #286: Use `huey` to schedule data ingestion
Data ingestion in the database can be very slow, specially for large datasets. Setting up a scheduler like `huey` for that purpose will improve the user experience. 

This should:
1. Add a `status` field and a `ingestion logs` field in the `DataImport` model that communicates the status of the ingestion to the user. 
2. Move most of the contents of `DataImport.clean` to a separate function that is setup as `@db_task` and that updates the 
3. Implement a `post_save` signal for the `DataImport` method that actually loads the data in the db asyncronously by calling this `@db_task` . This will update the fields indicated in 1 based on the status of the loading. 

Obviously, `huey` should be properly added as a dependency and configured.

### PR DESCRIPTION
**Pull request by dalonsoa**
This PR add the `huey` scheduler to ingest data asynchronously in the database, which results in a better user experience. 

A couple of fields have been added to the `DataImport` model (and the corresponding admin) to allow the user monitor the progress of the ingestion, as well as forcing the data to be re-processed, if needed.

An extra database, based on Sqlite had to be added as `huey` refuses to work with postgress.

Since I had to edit the `dockerfile` and `docker-compose` to launch the consumer, I have taken this opportunity to clean up the former, deleting some unnecessary, legacy files in the process. 

Close #286

### CODE REVIEW

**Comment by dalonsoa** [comment]
@cc-a , @AdrianDAlessandro , I'm going to start merging some Paricia PRs as I need to move forward with the development and make a release as soon as possible. Having said that, it will be good if, even after merging, you could have a look at this one as I'm not that familiar with `huey` and I would appreciate your feedback a lot in case I've done something stupid.

**Review by AdrianDAlessandro — COMMENTED** [review]
This seems sensible to me. I'm not super familiar with Huey, so I don't know about the specific choices you've made, but they seem reasonable.

File: importing/signals/handlers.py [hunk 1]
```diff
@@ -13,6 +14,7 @@
 def set_object_permissions(sender, instance: PermissionsBase, **kwargs):
     """Set object-level permissions"."""
     instance.set_object_permissions()
+    ingest_data(instance.pk)
```
Comment (cc-a) [inline importing/signals/handlers.py:17]: For clarity, I'd probably either put this in it's own appropriately named signal handler or rename this one.

File: docker-compose.yml [hunk 2]
```diff
@@ -16,13 +16,19 @@ services:
       - 5432:5432
   web:
     build: .
+    command:
+      - sh
+      - -c
+      - |
+        python manage.py migrate
+        python manage.py run_huey &
```
[the remark this answers is not in the document]
Comment (dalonsoa) [reply docker-compose.yml:24]: The settings I'm using for `huey` include `"immediate": False,` precisely so I can use scheduling in full even during development (thanks to that I discover that I needed to setup a separate DB for `huey` for it to schedule anything, as postgress was not good...`), so I need to call `run_huey`.

**Pull request merged**